.gitignore

```diff
@@ -1,7 +1,9 @@
+!openedx_webhooks/lib/
```

Comment:
I would remove this line from here, and the lib line from your global gitignore. Any repo that produces a lib directory as a build artifact should git-ignore it in the repo instead.